index.html

```diff
@@ -0,0 +1,194 @@
+                  >
+                  <label
+                    ><input
+                      name="prefer"
+                      value="data-visualization"
+                      type="checkbox"
+                      class="input-checkbox"
+                    />Data Visualization</label
+                  >
+                  <label
+                    ><input
+                      name="prefer"
+                      value="challenges"
+                      type="checkbox"
+                      class="input-checkbox"
+                    />Challenges</label
+                  >
+                  <label
+                    ><input
+                      name="prefer"
+                      value="open-source-community"
+                      type="checkbox"
+                      class="input-checkbox"
+                    />Open Source Community</label
+                  >
+                  <label
+                    ><input
+                      name="prefer"
+                      value="gitter-help-rooms"
+                      type="checkbox"
+                      class="input-checkbox"
+                    />Gitter help rooms</label
+                  >
+                  <label
+                    ><input
+                      name="prefer"
+                      value="videos"
+                      type="checkbox"
+                      class="input-checkbox"
+                    />Videos</label
+                  >
+                  <label
+                    ><input
+                      name="prefer"
+                      value="city-meetups"
+                      type="checkbox"
+                      class="input-checkbox"
+                    />City Meetups</label
+                  >
+                  <label
+                    ><input
+                      name="prefer"
+                      value="wiki"
+                      type="checkbox"
+                      class="input-checkbox"
+                    />Wiki</label
+                  >
+                  <label
+                    ><input
+                      name="prefer"
+                      value="forum"
+                      type="checkbox"
+                      class="input-checkbox"
+                    />Forum</label
+                  >
+                  <label
+                    ><input
+                      name="prefer"
+                      value="additional-courses"
+                      type="checkbox"
+                      class="input-checkbox"
+                    />Additional Courses</label
+                  >
+                  <textarea
+                  id="comments"
+                  class="input-textarea"
+                  name="comment"
+                  placeholder="Enter your comment here..."
+                ></textarea>
+
+                <button type="submit" id="submit" class="submit-button">
+                    Submit
+                  </button>
+
+          
+
+            
+
+        
```

Comment:
Using Linters are must, It would make your code more punctual.
If you would remove empty spaces will be better to make your code clear and redable.

Reply:
- [ ] great job giving reasons for requesting changes. please be kind and gentle when reviewing code. give suggestions instead of imposing changes on the person.

Reply:
If would use Linters, It would make your code more punctual.
Please remove all empty spaces to make your code clear and readable.